package.json

```diff
@@ -23,6 +24,6 @@
     "node": "*"
   },
   "scripts": {
-    "test": "node test/test.js"
+    "test": "node test/test.js || true"
```

Comment:
The added `|| true` means the test will simply never fail... Please revert this change as well.